src/components/Swap/Summary/index.tsx

```diff
@@ -287,23 +287,21 @@ export function SummaryDialog(props: SummaryDialogProps) {
   }, [ackPriceImpact, props.impact, showSpeedbump])
 
   return (
-    <>
+    <Column style={{ minWidth: isPageCentered ? Math.min(400, width) : 'auto', height: '100%' }} ref={setBoundary}>
```

Comment:
could something like this work?
```ts
{{ minWidth: isPageCentered ? 400 : 'auto', width: isPageCentered: '100%': undefined, height: '100%' }}
```

Reply:
also, is the 400 here related to the 420 width on speedbump?

Reply:
nope - this is just the min width of a page centered dialog. i've updated to name this value so it's more understandable.